app/src/main/java/com/glia/exampleapp/Application.java

```diff
@@ -43,35 +35,4 @@ private void initFirebase() {
                 .build();
         FirebaseApp.initializeApp(this, options);
     }
-
-    private void initGliaWidgets() {
-        registerActivityLifecycleCallbacks(new ActivityLifecycleCallbacks() {
-            @Override
-            public void onActivityCreated(@NonNull Activity activity, @Nullable Bundle savedInstanceState) {
-                if (activity.getClass() != com.glia.exampleapp.Activity.class) {
-                    if (Glia.isInitialized()) return;
-
-                    GliaWidgets.init(ExampleAppConfigManager.createDefaultConfig(getApplicationContext()));
-                }
-            }
-
-            @Override
-            public void onActivityStarted(@NonNull Activity activity) {}
-
-            @Override
-            public void onActivityResumed(@NonNull Activity activity) {}
-
-            @Override
-            public void onActivityPaused(@NonNull Activity activity) {}
-
-            @Override
-            public void onActivityStopped(@NonNull Activity activity) {}
-
-            @Override
-            public void onActivitySaveInstanceState(@NonNull Activity activity, @NonNull Bundle outState) {}
-
-            @Override
-            public void onActivityDestroyed(@NonNull Activity activity) {}
-        });
-    }
```

Comment:
As far as I see, this was needed to init Glia Widgets using a deep link. Are you sure it is not needed anymore?

Reply:
Yes, I moved this logic inside MainFragment and this way we have more control on it